### PR TITLE
Add more hints about template editing

### PR DESCRIPTION
**Pull request by greg0ire**
There are way too many people that don't seem to understand that this
template is supposed to be changed. Add more hints for them.

### CODE REVIEW

**Comment by greg0ire** [comment]
Not sure what more can be done… people just don't read sometimes.

File: project/.github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -51,6 +51,9 @@ Fixes #
 - [ ] Update the tests
 - [ ] Update the documentation
 - [ ] Add an upgrade note
+- [ ] Give a valid reason for targetting this branch
```
Comment (soullivaneuh) [inline project/.github/PULL_REQUEST_TEMPLATE.md:54]: Not a big fan about a todo for issue templating fill. We already have comment for that.

Plus, if the contributor didn't care of this template from the start, he will not care of this task list either.

**Comment by soullivaneuh** [comment]
You make a PR on the wrong repo. :-)

**Comment by greg0ire** [comment]
> You make a PR on the wrong repo. :-)

~~Sorry, how is this the wrong repo ?~~ Oh right, I see!

**Pull request closed without merging**